### ISSUE TITLE
Why don't you set styleganv2 model to eval()?

### ISSUE THREAD

**Issue by wty-ustc** [eladrich/pixel2style2pixel#61]
Dear sir：
  I have read your codes, it's a really good job. But I have a question, when training, why don't you set styleganv2 model to eval()?

**Comment by yuval-alaluf** [comment]
Good question. Since StyleGAN2 doesn't have any batch norm layers we do not need to set `eval()`, even during training. Note, however, that by default the parameters of the generator are not added to the optimizer and therefore the generator remains fixed during training.

**Comment by wty-ustc** [comment]
Thank you




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "eladrich/pixel2style2pixel"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月29日(星期二) 下午4:29
收件人:&nbsp;"eladrich/pixel2style2pixel"<pixel2style2pixel@noreply.github.com&gt;;
抄送:&nbsp;"wty"<845318759@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [eladrich/pixel2style2pixel] Why don't you set styleganv2 model to eval()? (#61)





 
Good question. Since StyleGAN2 doesn't have any batch norm layers we do not need to set eval(), even during training. Note, however, that by default the parameters of the generator are not added to the optimizer and therefore the generator remains fixed during training.
